.github/ISSUE_TEMPLATE.md

```diff
@@ -1,17 +0,0 @@
-**Describe the bug**
```

Comment:
I've removed the Issue and PR templates since I could not find a sufficiently consistent pattern in previous issues and PRs and thus I'd rather reduce the friction of creating new issues/PRs. Especially PR templates are a massive overhead every time you want to create a new PR.